Resources/config/serializer.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" ?>
+<container xmlns="http://symfony.com/schema/dic/services"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+
+    <services>
+        <service id="fos_rest.serializer.jms" class="FOS\RestBundle\Serializer\JmsSerializerAdapter" public="false" lazy="true">
+            <argument type="service" id="jms_serializer.serializer" />
```

Comment:
I think in this case the service should already be removed as it is an unused private service then (IIRC we have functional tests that cover this case).